package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-snippets",
-  "version": "0.8.0",
+  "version": "0.9.0",
```

Comment:
@rr-kalbeck can you roll this back? The publish script from atom's package manager automatically bumps this.

Reply:
ok, i rolled it backed